packages/backend/src/links/links.module.ts

```diff
@@ -1,9 +1,11 @@
-import { Module } from '@nestjs/common'
-import { LinksService } from './links.service'
-import { LinksController } from './links.controller'
+import { Module } from '@nestjs/common';
+import { LinksService } from './links.service';
+import { LinksController } from './links.controller';
+import { PrismaModule } from 'nestjs-prisma';
 
 @Module({
   controllers: [LinksController],
   providers: [LinksService],
+  imports: [PrismaModule],
```

Comment:
I think this won't be necessary, since it is imported globally in the app module (so it's available everywhere automatically) But I didn't try it, so not sure